web/apps/apps.go

```diff
@@ -107,10 +107,12 @@ func getHandler(appType consts.AppType) echo.HandlerFunc {
 		if err := middlewares.Allow(c, permission.GET, man); err != nil {
 			return err
 		}
+
+		registries := instance.Registries()
+		copier := app.Copier(consts.WebappType, instance)
```

Comment:
We'll need a konnector copier to update konnectors.

```suggestion
		copier := app.Copier(man.AppType(), instance)
```